From 7bfad75d6d86340cfcea031798ab6ee2aeeca438 Mon Sep 17 00:00:00 2001
From: Aravind Siddappaji <aravindx.siddappaji@intel.com>
Date: Sat, 15 Jun 2019 02:31:24 +0530
Subject: [PATCH] Debug patch for Volume & Mute Keyevent fix

Volume Up, Down & Mute Keyevents not effective
due to dependency on Car UI handling method
This is not a final fix, but suggestive debug patch to
arrive at final fix at UI handling volume keyevents

Change-Id: Ie4cfba800849364a59909409ccdbc678326d4582
Tracked-On: https://jira.devtools.intel.com/browse/OAM-76705
---
 services/core/java/com/android/server/media/MediaSessionService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/media/MediaSessionService.java b/services/core/java/com/android/server/media/MediaSessionService.java
index 68b2a58..0a69ee0 100644
--- a/services/core/java/com/android/server/media/MediaSessionService.java
+++ b/services/core/java/com/android/server/media/MediaSessionService.java
@@ -1455,7 +1455,7 @@ public class MediaSessionService extends SystemService implements Monitor {
                     if (up) {
                         flags |= AudioManager.FLAG_PLAY_SOUND | AudioManager.FLAG_VIBRATE;
                     } else {
-                        flags |= AudioManager.FLAG_SHOW_UI | AudioManager.FLAG_VIBRATE;
+                        flags |= /* AudioManager.FLAG_SHOW_UI | */ AudioManager.FLAG_VIBRATE;
                     }
                 }
                 if (direction != 0) {
-- 
2.7.4

